### PR TITLE
initial frame claims contracts

### PR DESCRIPTION
**Pull request by wwhchung**
Frame claims!

### CODE REVIEW

File: packages/manifold/contracts/frameclaims/FrameLazyClaim.sol [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    constructor(address initialOwner) {
+        _transferOwnership(initialOwner);
+    }
+
+    /**
+     * See {IFrameLazyClaim-setSigner}.
+     */
+    function setSigner(address signer) external override adminRequired {
```
Comment (campionfellin) [inline packages/manifold/contracts/frameclaims/FrameLazyClaim.sol:46]: Not needed

File: packages/manifold/contracts/frameclaims/FrameLazyClaim.sol [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        _signer = signer;
+    }
+
+    function _validateMint() internal {
+        if (msg.sender != _signer) revert InvalidSignature();
```
Comment (campionfellin) [inline packages/manifold/contracts/frameclaims/FrameLazyClaim.sol:51]: Oh, only manifold is allowed to mint?

File: packages/manifold/contracts/frameclaims/ERC1155FrameLazyClaim.sol [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+            interfaceId == type(IAdminControl).interfaceId ||
+            interfaceId == type(IERC165).interfaceId;
+    }
+
+    constructor(address initialOwner) FrameLazyClaim(initialOwner) {}
+
+    /**
+     * See {IERC1155LazyClaim-initializeClaim}.
+     */
+    function initializeClaim(
```
[the remark this answers is not in the document]
Comment (wwhchung) [reply packages/manifold/contracts/frameclaims/ERC1155FrameLazyClaim.sol:41]: Why? There are no price params anymore, cannot be paid.

[the remark this answers is not in the document]
Comment (wwhchung) [reply packages/manifold/contracts/frameclaims/ERC1155FrameLazyClaim.sol:41]: Not paid claims.

**Pull request merged**